rpcd/playbooks/roles/elasticsearch/defaults/main.yml

```diff
@@ -23,6 +23,9 @@ elasticsearch_legacy_apt_repos:
 # use -e 'logging_upgrade=true' to upgrade logging components
 logging_upgrade: false
```

Comment:
I think this var might be potentially dangerous. If ElasticSearch is 1.5 and this role is run with logging_upgrade set to false, ElasticSearch will be upgraded without the reindexing. Is there a situation where that would be appropriate? If this is going to be kept I think it would make sense to have the role fail in upgrade_pre_flight.yml if logging_upgrade is not true.